mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule ExTwilio.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/danielberkompas/ex_twilio"
-  @version "0.9.1"
+  @version "0.9.2"
```

Comment:
Why do we need to bump the version? As far as I know `.tool-versions` and `mix.lock` are both project-level files and have no bearing when the app is included in another project

Reply:
ahh i guess you're right. although it does make me wonder if we should not just update the mix.lock, but the required versions of those packages in the mix file?

Reply:
I guess I'm also not sure, does updating the packages in the mix lock actually mean when operator pulls in ex_twilio, without a version change, and without any actual version updates in the ex_twilio/mix.exs, will these updates in the mix.lock make their way into operator?

Reply:
So some of these deps upgrades are required for the latest erlang? I'm not sure the best way to notate that. We could update the requirements in mix.exs and have dependent apps upgrade to the new patch version when the upgrade their erlang version. Or we could just document that the transitive dependencies need upgraded.

But no, as far as I'm aware the `mix.lock` does not get inherited by dependent apps (and I believe isn't even included in a package).